ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/Account.java

```diff
@@ -39,6 +39,7 @@
   long DEFAULT_NONCE = 0L;
   Wei DEFAULT_BALANCE = Wei.ZERO;
   int DEFAULT_VERSION = 0;
+  int ISTANBUL_VERSION = 1;
```

Comment:
(Optional) Not sure about storing `ISTANBUL_VERSION` here - it feels very mainnet-evm-specific.  Looks like you could get away with just keeping this as a local variable in `MainnetProtocolSpecs`.

Reply:
Done